src/main/scala/io/iohk/ethereum/db/dataSource/EphemDataSource.scala

```diff
@@ -10,6 +10,10 @@ case class EphemDataSource(storage: Map[IndexedSeq[Byte], IndexedSeq[Byte]]) ext
       storage + ((namespace ++ toUpdate._1) -> toUpdate._2))
     EphemDataSource(afterUpdate)
   }
+
+  override def clear: DataSource = EphemDataSource(Map())
+
+  override def close(): Unit = Unit
```

Comment:
since this method does not behave as documented, maybe it's better to use `???` here